docs/releasing.md

```diff
@@ -81,7 +81,7 @@ We also need to create one or more tags for the Go modules ecosystem:
 
 - For any subdirectory with `go.mod` in it (excluding `hack/tools`), create
   another Git tag with directory prefix, ie.
-  `git tag apis/v0.x.y` and `git tag pkg/hardwareutils/v0.x.y`. This
+  `git tag apis/v0.x.y`, `git tag test/v0.x.y` and `git tag pkg/hardwareutils/v0.x.y`. This
```

Comment:
This line is 94 characters long. Can you make it under 80 line the rest of the doc?